client/src/context/types/detail.ts

```diff
@@ -0,0 +1,13 @@
+export interface IDialogItem {
+  title?: string;
+  content?: string;
+  icon?: string;
+  id?: string;
+}
+
+export interface IDialogItemContext {
+  items: IDialogItem[];
+  addItem: (item: IDialogItem) => boolean;
+  removeItem: (itemId: string) => void;
+  resetItems: () => boolean;
+}
```

Comment:
As I mentioned on Slack, moving these to the `interfaces` folder would be a better spot. Again, on me I started that `types` folder without noticing the other folder. 👍🏻

Reply:
Yep, makes sense. Just moved it.